src/ChatManager.js

```diff
@@ -1,7 +1,37 @@
 
+function parseJwt(token) {
+  var base64Url = token.split('.')[1];
+  var base64 = base64Url.replace(/-/g, '+').replace(/_/g, '/');
+  var jsonPayload = decodeURIComponent(atob(base64).split('').map(function(c) {
```

Comment:
even as this is succinct, is there a lib that could be used to do this instead of decomposing the token manually in this way?

such a lib would come in to use particularly if we were to change token formats, and the lib also supported the new token format.

Reply:
I contemplated about that. I decided to go this way since I didn't want to increase the size of the build by adding the entire auth0.js library. (unless there is something lightweight I am not aware of)

Reply:
I contemplated about that. I decided to go this way since I didn't want to increase the size of the build by adding the entire auth0.js library. (unless there is something lightweight I am not aware of)